src/runtime/virtcontainers/virtiofsd.go

```diff
@@ -34,13 +34,25 @@ var (
 	errVirtiofsdSocketPathEmpty    = errors.New("virtiofsd socket path is empty")
 	errVirtiofsdSourcePathEmpty    = errors.New("virtiofsd source path is empty")
 	errVirtiofsdSourceNotAvailable = errors.New("virtiofsd source path not available")
+
+	errUnimplemented = errors.New("unimplemented")
```

Comment:
These errors are grouped together, so the blank line should be removed.

Reply:
done